operator/avs_manager.go

```diff
@@ -285,6 +286,21 @@ func (avsManager *AvsManager) RegisterOperatorWithAvs(
 	return nil
 }
 
+func (avsManager *AvsManager) DeregisterOperator(blsKeyPair *bls.KeyPair,) (error) {
+	// TODO: 'QuorumNums' is hardcoded for now
+	quorumNumbers := eigentypes.QuorumNums{0}
+	pubKey := eigenutils.ConvertToBN254G1Point(blsKeyPair.GetPubKeyG1())
+
+	_, err := avsManager.avsWriter.DeregisterOperator(context.Background(), quorumNumbers, pubKey)
+	if err != nil {
+		avsManager.logger.Errorf("Unable to deregister operator with avs registry coordinator")
```

Comment:
Would be good to print error as well

Reply:
Now we do: we propagate the error to `main.go` (see https://github.com/NethermindEth/near-sffl/pull/171/files#diff-b8b904d92cea98f8256ea3b62d416f66151d1315c5b58d870bfb553877724be4R201) and log the error in case there is any